masterlist.yaml

```diff
@@ -2076,6 +2076,27 @@ plugins:
       - crc: 0x1DC92B96
         util: 'TES4Edit v4.0.4b'
 
+  - name: 'Bloodworm_Helm_(Hornless|Replacer)\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/oblivion/mods/35544/'
+        name: 'Bloodworm Helm Replacer'
+    msg:
+      - <<: *useOnlyOneX
+        subs: [ 'Bloodworm Helm Replacer ESP' ]
+        condition: 'many("Bloodworm_Helm_(Hornless|Replacer)\.esp")'
+    tag:
+      - Deactivate
+      - Graphics
+      - NoMerge
+  - name: 'Bloodworm_Helm_Hornless.esp'
+    clean:
+      - crc: 0xC29EA8C9
+        util: 'TES4Edit v4.0.4b'
+  - name: 'Bloodworm_Helm_Replacer.esp'
```

Comment:
Since the ESPs are identical in all but name, the cleaning data can be moved to the regex entry.

```suggestion
```